topology/Fermi National Accelerator Laboratory/FNAL USCMS Tier1/USCMS-FNAL-WC1.yaml

```
@@ -405,5 +405,24 @@ Resources:
       KSI2KMin: 0
       StorageCapacityMax: 24000
       StorageCapacityMin: 24000
-      TapeCapacity: 0
+      TapeCapacity: 0      
+  cmsfrontier_squid:
```

Suggestion for the resource name to match the format of the other resources in the file:

```suggestion
  USCMS-FNAL-WC1-SQUID:
```